frontend/src/org/components/EventForm/steps/BasicInfoStep.tsx

```diff
@@ -201,9 +201,12 @@ export const BasicInfoStep = ({
                                           name={field.name}
                                           id={slug}
                                           value={id}
-                                          checked={field.value?.includes(id)}
+                                          checked={(
+                                        field.value as number[]
+                                      )?.includes(id)}
                                           onChange={e => {
-                                            // check when unchecked and vise-versa
+                                            if (typeof field.value === 'number') {
+                                        // check when unchecked and vise-versa
```

Comment:
The alignment of these lines is a bit off, everything in the "if" block should be two spaces more to the right than the "if" line. Can you please fix that?